src/macros.rs

```diff
@@ -20,6 +20,13 @@ macro_rules! define_uuid_macro {
                 };
                 OUTPUT
             }};
+            ($uuid:ident) => {{
```

Comment:
```suggestion
            ($uuid:expr) => {{
```

We can generalize this to match any expression and it will still work.